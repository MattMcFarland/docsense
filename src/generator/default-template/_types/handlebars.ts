export interface BlockHelper {
  fn: (ctx: any) => any;
}
