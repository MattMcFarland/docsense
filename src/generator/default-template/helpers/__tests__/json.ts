import * as Handlebars from 'handlebars';

describe('hbs helpers > json', () => {
  test('matches snapshot', () => {
    Handlebars.registerHelper('json', require('../json').default);
    const template = Handlebars.compile(`<div>{{json pojo}}</div>`);
    const result = template({ pojo: { foo: { bar: 'baz ' } } });
    expect(result).toMatchSnapshot();
  });
});
