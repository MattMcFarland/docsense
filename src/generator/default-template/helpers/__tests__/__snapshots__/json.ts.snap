// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hbs helpers > json matches snapshot 1`] = `
"<div>{
  &quot;foo&quot;: {
    &quot;bar&quot;: &quot;baz &quot;
  }
}</div>"
`;
